src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveUtil.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring.move.common
+
+import com.intellij.psi.PsiComment
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import org.rust.ide.inspections.import.insertUseItem
+import org.rust.lang.core.psi.RsBlock
+import org.rust.lang.core.psi.RsPath
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.psi.RsUseItem
+import org.rust.lang.core.psi.ext.*
+
+class RsMoveReferenceInfo(
+    val pathOld: RsPath,
+    val pathNew: RsPath?,
+    // == `pathOld.reference.resolve()`
+    val target: RsQualifiedNamedElement
+) {
+    val isInsideUseDirective: Boolean get() = pathOld.ancestorStrict<RsUseItem>() != null
+}
+
+fun RsPath.isAbsolute(): Boolean {
+    if (text.startsWith("::")) return true
```

Comment:
```suggestion
    if (hasColonColon) return true
```

Reply:
Sorry, I don't understand this change. Consider path `foo::bar`, it has `hasColonColon == true`, but if `foo` is not crate root, then this path is not absolute

Reply:
For given path `foo::bar` `hasColonColon == false` :woman_shrugging: 
`hasColonColon` is almost equivalent to `text.startsWith("::")`

Reply:
I thought that `hasColonColon` is equivalent to `text.contains("::")`, isn't it?

Maybe I do something wrong, but on the screenshot `hasColonColon` is true for path `mod1::foo`:
![image](https://user-images.githubusercontent.com/6505554/87143963-6dcecc00-c2c0-11ea-8468-84b8f1395250.png)

Reply:
Oh, I'm you're right, I'm sorry. The, `basePath().hasColonColon` should work

Reply:
Ok, so replaced with `basePath().hasColonColon`, thank you